src/components/UsernameEntry/UsernameEntry.js

```diff
@@ -4,9 +4,10 @@ import SlackIcon from './SlackIcon';
 import './UsernameEntry.css';
 
 class UsernameEntry extends React.Component {
-	sendReminder = () => {
-		this.props.sendSlackMessage(this.props.currentUser);
-		// handle this.props.sendReminderError
+	sendReminder = async () => {
```

Comment:
this change appears to be nothing to do with the styles, please remember to keep your PRs small and clean